build.rs

```diff
@@ -329,9 +332,15 @@ fn zip_dylint_driver(src_dir: &Path, dst_file: &Path, method: CompressionMethod)
             buffer.clear();
 
             zip.finish()?;
+            lib_found = true;
             break;
         }
     }
+
+    if !lib_found {
+        anyhow::bail!("Couldn't find compiled lint. Is your architecture defined in ./ink_linting/.cargo/config.toml?");
```

Comment:
Could you include the architecture here in the bail message? Makes debugging easier.